app/views/layouts/application.html.erb

```diff
@@ -13,6 +13,8 @@
   </head>
 
   <body>
+    <p class="notice"><%= notice %></p>
+    <p class="alert"><%= alert %></p>
```

Comment:
- [ ] It will be better to add links for sign-in and sign-up, so the user can use them to navigate to the registration form and consider adding a logout button.

```ruby
<% if user_signed_in? %>
        <%= button_to "Log Out", destroy_user_session_path, method: :delete %>
<% else %>
         <%= link_to "Sign In", new_user_session_path, method: :delete %>
         <%= link_to "Sign Up", new_user_registration_path, method: :delete %>
 <% end %>
```